characters-attack.html

```diff
@@ -1,44 +1,36 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="pl">
 
 <head>
     <meta charset="utf-8">
     <title>OVERWATCH - POSTACIE NATARCIA</title>
-    <meta name="keywords" content="owerwatch, owerwatch postacie, gry strzelanie" />
+    <meta name="keywords" content="overwatch, overwatch heroes, game shooting" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"  />
     <link rel="stylesheet" href="style.css" type="text/css" />
-
 </head>
 
 <body>
-    <div id="container">
-        <div id="logo">
+    <section class="container">
+        <div id="top-title">
```

Comment:
change it to `class`, you have more than one `id="top-title"`. And is used to styling (we have to avoid such an approach)